lib/util/Opus.js

```diff
@@ -0,0 +1,29 @@
+let NodeOpus;
+try {
+    NodeOpus = require("node-opus");
+} catch(err) { // eslint-disable no-empty
+}
+let OpusScript;
+try {
+    OpusScript = require("opusscript");
+} catch(err) { // eslint-disable no-empty
+}
+
+
+module.exports.createOpus = function createOpus (samplingRate, channels, bitrate) {
```

Comment:
```suggestion
module.exports.createOpus = function createOpus(samplingRate, channels, bitrate) {
```

Reply:
Consistency. See `pickCommand` on VoiceConnection

Reply:
I am not very confident about what did you mean.

Did you means some structure like

```js
module.exports.opusFactory  = opusFactory = {
  impl: null,
  selectImpl () {
    // whatevet...
  },
  create () {
     if (this.impl == null) this.selectImpl()
    // whatevet...
     return opus
  }
}
```

?

Reply:
I just meant that the other time there is something like `= function x(` there is no space between the name and the (
![image](https://user-images.githubusercontent.com/23035000/72349219-17365b80-36aa-11ea-8d41-6677264a35d0.png)

Reply:
ok, fixed